### PR TITLE
Add note about the Alby Hub and redirect users to the hub

### PR DESCRIPTION
**Pull request by bumi**
nwc.getalby.com gets depreacted

### CODE REVIEW

File: echo_handlers.go [hunk 1]
```diff
@@ -284,6 +284,9 @@ func (svc *Service) AppsNewHandler(c echo.Context) error {
 		sess.Save(c.Request(), c.Response())
 		return c.Redirect(302, fmt.Sprintf("/%s/auth?c=%s", strings.ToLower(svc.cfg.LNBackendType), appName))
 	}
+	if user.HubUrl != "" {
+		return c.Redirect(302, fmt.Sprintf("%s/#/apps/new%s", user.HubUrl, c.QueryString()))
```
Comment (rolznz) [inline echo_handlers.go]:
```suggestion
		return c.Redirect(302, fmt.Sprintf("%s/#/apps/new?%s", user.HubUrl, c.QueryString()))
```

File: migrations/202407110000_add_user_hub_url.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+// Create a composite index to improve performance of finding the latest nostr event for an app
```
Comment (rolznz) [inline migrations/202407110000_add_user_hub_url.go]:
```suggestion
// save hub URL so it only has to be requested once on auth
```

**Comment by rolznz** [comment]
@bumi updated banner style a bit, + whole banner is clickable

image

File: echo_handlers.go (hunk 1 above)
Comment (bumi) [reply echo_handlers.go]: hmm. actually I was wondering here. thanks for double checking

**Comment by bumi** [comment]
oh nice!

File: alby.go [hunk 3]
```diff
@@ -659,7 +659,7 @@ func (svc *AlbyOAuthService) CallbackHandler(c echo.Context) error {
 	}
 	client := svc.oauthConf.Client(c.Request().Context(), tok)
 
-	req, err := http.NewRequest("GET", fmt.Sprintf("%s/user/me", svc.cfg.AlbyAPIURL), nil)
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s/internal/users", svc.cfg.AlbyAPIURL), nil)
```
Comment (im-adithya) [inline alby.go:662]: ⚠️ Have to check if the `ALBY_CLIENT_SECRET` is the default in deployment repo

**Review by im-adithya — APPROVED** [review]

File: alby.go (hunk 3 above)
Comment (bumi) [reply alby.go:662]: what do you mean?

Comment (im-adithya) [reply alby.go:662]: We can only make calls to the internal api only using the internal client right?

**Pull request merged**